framework/helpers/BaseArrayHelper.php

```diff
@@ -999,4 +999,19 @@ public static function filter($array, $filters)
 
         return $result;
     }
+
+    /**
+     * Sorts multidimensional array
+     * @param $array
+     * @return void
+     */
+    public static function sortMultidimensionalArray(&$array)
+    {
+        ksort($array);
```

Comment:
sorry, but it's work only for assoc arrays, universal solution:
~~~php
if (static::isIndexed($array)) {
   sort($array);
} else {
   ksort($array);
}
~~~

Reply:
Why? ksort works for int-indexed arrays.

Reply:
@bizley index array - sort by values, assoc array - sort by keys:
~~~php
$oldValue = ['foo' => ['bar' => 1, 'baz' => 2]];
$newValue = ['foo' => ['baz' => 2, 'bar' => 1]];

$oldValue = [['bar' => 1], ['baz' => 2]];
$newValue = [['baz' => 2], ['bar' => 1]];
~~~

Reply:
Yes, I know. @speedplli please elaborate on this step here.

Reply:
and add tests

Reply:
@bizley I will.
@WinterSilence Thanks I will implement.

Reply:
@bizley @WinterSilence 
Done Please check. 
Tests were also added.

Thanks